flask/src/server.py

```diff
@@ -0,0 +1,39 @@
+import ffmpeg
+import tempfile
+import os
+
+from girder_client import GirderClient
+from flask import Flask, request, Response, send_file
+from flask_cors import CORS
+
+app = Flask(__name__)
+
+cors_domain = os.environ.get('CORS_API', '*')
```

Comment:
Sorry to be a pain but the env var should probably be CORS_DOMAIN as well.

Reply:
That makes sense, sorry for overlooking that. Fixed.